includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -244,14 +244,14 @@
 		<tr>
 			<td class="label"><label for="refund_amount"><?php _e( 'Refund amount', 'woocommerce' ); ?>:</label></td>
 			<td class="total">
-				<input type="text" class="text" id="refund_amount" name="refund_amount" class="wc_input_price" />
+				<input type="text" class="wc_input_strict_price" id="refund_amount" name="refund_amount" />
 				<div class="clear"></div>
 			</td>
 		</tr>
 		<tr>
 			<td class="label"><label for="refund_reason"><?php echo wc_help_tip( __( 'Note: the refund reason will be visible by the customer.', 'woocommerce' ) ); ?> <?php _e( 'Reason for refund (optional):', 'woocommerce' ); ?></label></td>
 			<td class="total">
-				<input type="text" class="text" id="refund_reason" name="refund_reason" />
+				<input type="text" class="wc_input_strict_price" id="refund_reason" name="refund_reason" />
```

Comment:
The class `wc_input_strict_price` doesn't make sense here. The input is for reason of refund.